File: Ejemplos/43-DockerAndTestContainers/src/main/java/dev/joseluisgs/model/Persona.java
package dev.joseluisgs.model;

import lombok.Builder;
import lombok.Data;

import java.time.LocalDateTime;
import java.util.UUID;

@Data
@Builder
public class Persona {
    private long id;
    private UUID uuid;
    private String nombre;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: Ejemplos/33-EjemplosTestRepositorioServicio/src/main/kotlin/personas/model/Persona.kt
package dev.joseluisgs.personas.model

import java.time.LocalDateTime
import java.util.*

data class Persona(
    val ID: UUID = UUID.randomUUID(),
    val nombre: String,
    val edad: Int,
    val createdAt: LocalDateTime = LocalDateTime.now(),
    val updatedAt: LocalDateTime = LocalDateTime.now()
)